quadriga/src/main/java/edu/asu/diging/quadriga/core/data/CollectionRepository.java

```diff
@@ -8,4 +9,6 @@
 @Repository
 public interface CollectionRepository extends MongoRepository<Collection, String>{
 
+    Collection findBy_id(ObjectId _id);
```

Comment:
I would change this repo to `MongoRepository<Collection, ObjectId>` and annotate `id` in Collection with `@Id`. You shouldn't need this method then.